proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -80,11 +100,135 @@ message Event {
   // Payload of the audited event. It specifies additional context on the event.
   oneof payload {
     PayloadOrganizationCreated organization_created = 6;
+    PayloadOrganizationDeleted organization_deleted = 7;
+    PayloadOrganizationMemberAdded organization_member_added = 8;
+    PayloadOrganizationMemberRoleChanged organization_member_role_changed = 9;
+    PayloadOrganizationMemberRemoved organization_member_removed = 10;
+    PayloadRepositoryCreated repository_created = 11;
+    PayloadRepositoryDeleted repository_deleted = 12;
+    PayloadRepositoryPushed repository_pushed = 13;
+    PayloadRepositoryMemberAdded repository_member_added = 14;
+    PayloadRepositoryMemberRoleChanged repository_member_role_changed = 15;
+    PayloadRepositoryMemberRemoved repository_member_removed = 16;
+    PayloadPluginCreated plugin_created = 17;
+    PayloadPluginDeleted plugin_deleted = 18;
+    PayloadPluginPushed plugin_pushed = 19;
+    PayloadUserRegistered user_registered = 20;
+    PayloadUserDeactivated user_deactivated = 21;
+    PayloadUserDeleted user_deleted = 22;
+    PayloadUserLoggedIn user_logged_in = 23;
+    PayloadUserLoggedOut user_logged_out = 24;
   }
 }
 
-// PayloadOrganizationCreated is the payload for an ORGANIZATION_CREATED event.
 message PayloadOrganizationCreated {
-  // created_time is when the organization was created.
-  google.protobuf.Timestamp created_time = 1;
+  // create_time is when the organization was created.
+  google.protobuf.Timestamp create_time = 1;
 }
+
+message PayloadOrganizationDeleted {}
+
+message PayloadOrganizationMemberAdded {
+  // member_id is the id of the member added to the organization.
+  string member_id = 1;
+  // member_name is the name of the member added to the organization.
+  string member_name = 2;
+  // new_role is the role granted to the member added to the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole new_role = 3;
+}
+
+message PayloadOrganizationMemberRoleChanged {
+  // member_id is the id of the member whose role was changed.
+  string member_id = 1;
+  // member_name is the name of the member whose role was changed.
+  string member_name = 2;
+  // old_role is the old role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.OrganizationRole old_role = 3;
+  // new_role is the new role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.OrganizationRole new_role = 4;
+}
+
+message PayloadOrganizationMemberRemoved {
+  // member_id is the id of the member removed from the organization.
+  string member_id = 1;
+  // member_name is the name of the member removed from the organization.
+  string member_name = 2;
+  // old_role is the old role of the member removed from the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole old_role = 3;
+}
+
+message PayloadRepositoryCreated {
+  // create_time is when the repository was created.
+  google.protobuf.Timestamp create_time = 1;
+}
```

Comment:
You can add here the `owner_name` and `owner_id`. It could be different from the actor, it could be an organization.